prime-router/build.gradle.kts

```diff
@@ -497,9 +499,22 @@ tasks.azureFunctionsPackage {
     finalizedBy("copyAzureScripts")
 }
 
+tasks.register("generateVersionFile") {
+    doLast {
+        val stdout = ByteArrayOutputStream()
+        exec {
+            commandLine("git", "rev-parse", "--short", "HEAD")
+            standardOutput = stdout
+        }
+        val currentCommit = stdout.toString(StandardCharsets.UTF_8)
+        File("./version.txt").writeText("Commit ID: $currentCommit")
```

Comment:
I would write this file as a properties/json/yaml file (anything easily machine readable).

Reply:
Good point, changed it to a json file